### PR TITLE
Week_1_Task

### PR DESCRIPTION
**Pull request by ssinha2103**

### CODE REVIEW

File: UP_onboarding_system/UP_onboarding_system/settings.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+WSGI_APPLICATION = 'UP_onboarding_system.wsgi.application'
+
+# Database
+# https://docs.djangoproject.com/en/4.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'up-project',
```
Comment (abhi-1408) [inline UP_onboarding_system/UP_onboarding_system/settings.py:83]: Can u try using these `user`, `password` etc from the `.env` itself.

Comment (ssinha2103) [reply UP_onboarding_system/UP_onboarding_system/settings.py:83]: Sure, on it.

File: UP_onboarding_system/point_of_sale/models.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    def __str__(self):
+        return "{}: {}".format(self.name, self.address)
+
+    class Meta:
+        verbose_name_plural = 'Stores'
+
+
+class Items(models.Model):
+    name = models.CharField(max_length=50, unique=True)
```
Comment (abhi-1408) [inline UP_onboarding_system/point_of_sale/models.py]: for name of item we have `unique=True` here, can u please explain why did u add it. 
We might have same items in different stores.

File: UP_onboarding_system/point_of_sale/models.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
```
Comment (abhi-1408) [inline UP_onboarding_system/point_of_sale/models.py:4]: Also please add any more fields that u feel can be added in the below models.

File: UP_onboarding_system/point_of_sale/models.py (hunk 2 above)
Comment (ssinha2103) [reply UP_onboarding_system/point_of_sale/models.py]: According to me, As this is linked with Store , the unique property is of the items with 1 store . We can create same item in different store .

File: UP_onboarding_system/point_of_sale/models.py (hunk 3 above)
Comment (ssinha2103) [reply UP_onboarding_system/point_of_sale/models.py:4]: Ok sure, I added only those which were mentioned in the onboarding guide. I will add few more .

File: UP_onboarding_system/point_of_sale/models.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+class Profile(models.Model):
+    MERCHANT = 1
+    CONSUMER = 2
+
+    ROLE_CHOICES = (
+        (MERCHANT, 'MERCHANT'),
+        (CONSUMER, 'CONSUMER')
+    )
+
+    name = models.CharField(max_length=50)
```
Comment (abhi-1408) [inline UP_onboarding_system/point_of_sale/models.py:14]: Also we already have `first_name` & `last_name` on User model, do we need additional field `name` on `Profile`

File: UP_onboarding_system/point_of_sale/tests/tests.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    # import pdb; pdb.set_trace()
+    response = client.post("/login/", dict(username="consumer", password="password@123"))
+    token = response.data["access"]
+    store = Stores.objects.create(name="WhiteField Outlet", address="WhiteField", lat=13, lng=13,
+                                  merchant=merchant_data)
+    item = Items.objects.create(name="Dosa", price=120, description="Sambhar Dosa", stores=store)
+    req_send = client.post("/placeorders/",
+                           dict(user=consumer_data, merchant=merchant_data.pk, store=store.pk, items=item.pk),
+                           **{'HTTP_AUTHORIZATION': f'Bearer {token}'})
+    assert req_send.status_code == 201
```
Comment (abhi-1408) [inline UP_onboarding_system/point_of_sale/tests/tests.py:80]: Also can u assert if the obj was created in the model with the details provided.

Comment (abhi-1408) [reply UP_onboarding_system/point_of_sale/tests/tests.py:80]: *obj of the respective model

Comment (abhi-1408) [inline UP_onboarding_system/point_of_sale/tests/tests.py:83]: Also can u assert if the obj was created for the respective model with the details provided.

File: UP_onboarding_system/UP_onboarding_system/settings.py (hunk 1 above)
Comment (ssinha2103) [reply UP_onboarding_system/UP_onboarding_system/settings.py:83]: Done.

File: UP_onboarding_system/point_of_sale/models.py (hunk 4 above)
Comment (ssinha2103) [reply UP_onboarding_system/point_of_sale/models.py:14]: I am using User model for merchant and consumer profile , so for merchants it easier to have 1 name , that's why I did it like that.

File: UP_onboarding_system/point_of_sale/tests/tests.py (hunk 5 above)
Comment (ssinha2103) [reply UP_onboarding_system/point_of_sale/tests/tests.py:83]: Sure , will do that.

Comment (ssinha2103) [reply UP_onboarding_system/point_of_sale/tests/tests.py:83]: Done.

**Comment by abhi-1408** [comment]
Good job @ssinha2103,
As we discussed on call there are a few changes ( based on priority ) that can be taken up:

1. modify password view
2. reseting password
3. fetch user List
4. merchant name return in list
5. reseting of form after posting it
6. return store name in place of id
7. unique false in item name (unique together)
8. filter out items in place order
9. items check in test with filter

**Comment by ssinha2103** [comment]
> Good job @ssinha2103, As we discussed on call there are a few changes ( based on priority ) that can be taken up:
> 
> 1. modify password view
> 2. reseting password
> 3. fetch user List
> 4. merchant name return in list
> 5. reseting of form after posting it
> 6. return store name in place of id
> 7. unique false in item name (unique together)
> 8. filter out items in place order
> 9. items check in test with filter

@abhi-1408 I have incorporated the changes, as such I am merging the PR.

**Pull request merged**